src/sentry_plugins/github/plugin.py

```diff
@@ -228,7 +228,10 @@ def get_configure_plugin_fields(self, request, project, **kwargs):
             'default': self.get_option('repo', project),
             'type': 'text',
             'placeholder': 'e.g. getsentry/sentry',
-            'help': 'Enter your repository name, including the owner.',
+            'help': ('Enter your repository name, including the owner. '
+                    '<p><b>Looking for commit data?</b> You\'ll need to configure this through our'
```

Comment:
i'd maybe change this to something like "Looking to integrate commit data with releases?..."